src/vmm/src/io_uring/bindings.rs

```diff
@@ -89,7 +89,7 @@ pub struct io_uring_sqe {
     pub __bindgen_anon_2: io_uring_sqe__bindgen_ty_2,
     pub len: __u32,
     pub __bindgen_anon_3: io_uring_sqe__bindgen_ty_3,
-    pub user_data: __u64,
+    pub user_data_key: usize,
```

Comment:
Is it fine to update bindings that are automatically generated? Doesn't this result in requiring manual change every time we update bindings in line with the newer kernel in the future?

Reply:
oh I missed that this is autogenerated, I'll revert the change. Thanks for noticing it!